### ISSUE TITLE
[1.10.2] Item texture not working

### ISSUE THREAD

**Issue by bortooo** [tterrag1098/CustomThings#56]
Custom item texture don't work
Model: [bedrockiumingot.json](https://pastebin.com/FjDtQDrT)
Config: [customItem.json](https://pastebin.com/e4xe01mj)

**Comment by gdarai** [comment]
I have to repot the same issue. Also, when I list all items by minetweaker, I am reading strange behaviour:
My new Item in customItems.json  "name":"coin" is registered in minetweaker as <customitems:item> isn't it strange? BTW, the version is also 1.10.2

**Comment by gdarai** [comment]
To add to this topic,
when I add second item,
mine tweaker sees only the first one, even those minecraft has both registered.
Adding the customIems.json to be clear:
```
{"data":
	[
		{
			"name":"coin",
			"rarity":"rare",
			"enchanted":true
		},
		{
			"name":"wood_rod",
			"rarity":"common",
			"enchanted":false
		}
	]
}
```

**Comment by tterrag1098** [comment]
that behavior is normal, *all* basic items in CustomThings share the same item object, they just use different damage values. To specify them in MT you need to use damage, so it would be <customthings:item:1> for instance.

**Comment by tterrag1098** [comment]
Fixed via 08df8cb9ebed52f0bf0354b89d2d7ec1710161fe
